SironaMedical/philter-lite
# AIE-2493 py311 and other packaging updates

**Fixes:** [AIE-2493](https://sironamedical.atlassian.net/browse/AIE-2493)

## Summary

* add support for py311 (testing, etc)
* drop support for py37
* update outdated dependencies
* add details to pyproject so that they appear in the pypi page upon package release
* update the mechanism for setting the `__version__` so that it is correct, and based on the pyproject setting

[AIE-2493]: https://sironamedical.atlassian.net/browse/AIE-2493?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
